<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="hu-hu">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">3b57045d08ea0a16088d5cd2fc0ae01f4bee9e74</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.Xaml.XamlSchemaException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">b34b2e30a09bcee2aa0796fb648dc7d48ad5768f</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">9dbe9cfd6004d91e84edc43f47fb05050674de5b</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>The exception that is thrown when a binding system or another schema representation system for XAML reports an exception to the schema context.</source>
          <target state="translated">A kivétellel, hogy amikor egy kötés vagy egy másik séma ábrázolását és XAML jelzi a séma környezetében kivételt vált ki.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT">
          <source>The XamlSchemaException is thrown by several cases in the CLR binding support classes that provide the schema context for .NET Framework XAML Services (&lt;xref:System.Xaml.XamlXmlReader&gt;, &lt;xref:System.Xaml.XamlObjectWriter&gt;, or &lt;xref:System.Xaml.XamlServices&gt;).</source>
          <target state="translated">A XamlSchemaException fordul elő a CLR-beli kötés támogatási osztályok, amelyek a séma környezetében .NET-keretrendszer XAML-szolgáltatások számos esetben által (&lt;xref:System.Xaml.XamlXmlReader&gt;, &lt;xref:System.Xaml.XamlObjectWriter&gt;, vagy &lt;xref:System.Xaml.XamlServices&gt;).&lt;/xref:System.Xaml.XamlServices&gt; &lt;/xref:System.Xaml.XamlObjectWriter&gt; &lt;/xref:System.Xaml.XamlXmlReader&gt;</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT">
          <source>A XamlSchemaException is typically passed as the inner exception for other exceptions, such as the &lt;xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=fullName&gt;-true property for a &lt;xref:System.Xaml.XamlObjectWriterException&gt;, so it should be uncommon to catch a XamlSchemaException directly unless you are implementing a component such as a specialized object writer.</source>
          <target state="translated">Egy XamlSchemaException általában átadása más kivételeket, a belső kivétel üzenetében, például a &lt;xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=fullName&gt;-tulajdonsága true egy &lt;xref:System.Xaml.XamlObjectWriterException&gt;, ezért közvetlenül tényleges egy XamlSchemaException, kivéve, ha egy összetevő, például egy speciális objektum író webkiszolgálókból ritka kell lennie.&lt;/xref:System.Xaml.XamlObjectWriterException&gt; &lt;/xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=fullName&gt;</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Xaml.XamlSchemaException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a system-supplied message that describes the error.</source>
          <target state="translated">Egy új példányát inicializálja a <bpt id="p1">&lt;xref href="System.Xaml.XamlSchemaException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> osztály rendszer által megadott üzenetet, amely leírja a hibát.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Xaml.XamlSchemaException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified message that describes the error.</source>
          <target state="translated">Egy új példányát inicializálja a <bpt id="p1">&lt;xref href="System.Xaml.XamlSchemaException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> osztály egy adott üzenet, amely leírja a hibát.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve">
          <source>The message that describes the exception.</source>
          <target state="translated">A kivétel leíró üzenet.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Xaml.XamlSchemaException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with serialized data.</source>
          <target state="translated">Egy új példányát inicializálja a <bpt id="p1">&lt;xref href="System.Xaml.XamlSchemaException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> a szerializált adatokban osztályt.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve">
          <source>The object that holds the serialized object data.</source>
          <target state="translated">A szerializált objektum adatokat tartalmazó objektum.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source>The contextual information about the source or destination.</source>
          <target state="translated">A cél- és környezeti információt.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Xaml.XamlSchemaException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Egy új példányát inicializálja a <bpt id="p1">&lt;xref href="System.Xaml.XamlSchemaException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> megadott hibaüzenet, és hivatkozni kell a belső kivételben találhatók okozza-e ezt a kivételt, a osztályt.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.</source>
          <target state="translated">Kivételt vált ki, mert egy előző kivétel közvetlen eredménye tartalmaznia kell egy hivatkozást az előző kivétel az a &lt;xref:System.Exception.InnerException%2A&gt;tulajdonság.&lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or <ph id="ph1">`null`</ph> if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.</source>
          <target state="translated">A &lt;xref:System.Exception.InnerException%2A&gt;tulajdonság a konstruktornak átadott ugyanazt az értéket adja vissza vagy <ph id="ph1">`null`</ph> Ha a &lt;xref:System.Exception.InnerException%2A&gt;tulajdonság nem adja meg a belső kivétel értékét a konstruktor.&lt;/xref:System.Exception.InnerException%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source>The message that describes the exception.</source>
          <target state="translated">A kivétel leíró üzenet.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">A kivétellel, hogy a jelenlegi kivétel okozza.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve">
          <source>If the <ph id="ph1">`innerException`</ph> parameter is not <bpt id="p1">&lt;xref uid="langword_csharp_null" name="null" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>, the current exception is raised in a <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept> block that handles the inner exception.</source>
          <target state="translated">Ha a <ph id="ph1">`innerException`</ph> paraméter nem <bpt id="p1">&lt;xref uid="langword_csharp_null" name="null" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>, a jelenlegi kivétel a következik be egy <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt> <ept id="p2">&lt;/xref&gt;</ept> blokk, amely kezeli a belső kivételben találhatók.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>